### ISSUE TITLE
Unable to run juju bootstrap on Azure Ubuntu VM

### ISSUE THREAD

**Issue by activatedgeek** [juju/juju#2317]
I have configured the `environments.yaml` file as per the documentation. But I get the following error on running `juju bootstrap -e azure -v`. (I had also made the `azure` environment default as a double check).

```
Bootstrap failed, cleaning up the environment.
ERROR there was an issue examining the environment: cannot acquire management API: crypto/tls: failed to parse key: asn1: structure error: tags don't match (2 vs {class:0 tag:16 length:976 isCompound:true}) {optional:false explicit:false application:false defaultValue:<nil> tag:<nil> stringType:0 set:false omitEmpty:false} int @4
```

Here is my `environments.yaml`:
```
default: azure 

environments:
    azure:
        type: azure
    
        # location specifies the place where instances will be started,
        # for example: West US, North Europe.
        #
        location: Southeast Asia 
    
        # The following attributes specify Windows Azure Management
        # information. See:
        # http://msdn.microsoft.com/en-us/library/windowsazure
        # for details.
        #
        management-subscription-id: <subscription-id-here>
        management-certificate-path: /home/juju/file.pem
    
        # storage-account-name holds Windows Azure Storage info.
        #
        storage-account-name: <storage-account-name>
    
        # force-image-name overrides the OS image selection to use a fixed
        # image for all deployments. Most useful for developers.
        #
        # force-image-name: b39f27a8b8c64d52b05eac6a62ebad85__Ubuntu-13_10-amd64-server-DEVELOPMENT-20130713-Juju_ALPHA-en-us-30GB
    
        # image-stream chooses a simplestreams stream from which to select
        # OS images, for example daily or released images (or any other stream
        # available on simplestreams).
        #
        # image-stream: "released"
    
        # agent-stream chooses a simplestreams stream from which to select tools,
        # for example released or proposed tools (or any other stream available
        # on simplestreams).
        #
        # agent-stream: "released"
    
        # Whether or not to refresh the list of available updates for an
        # OS. The default option of true is recommended for use in
        # production systems, but disabling this can speed up local
        # deployments for development or testing.
        #
        # enable-os-refresh-update: true
    
        # Whether or not to perform OS upgrades when machines are
        # provisioned. The default option of true is recommended for use
        # in production systems, but disabling this can speed up local
        # deployments for development or testing.
        #
        # enable-os-upgrade: true
```

**Comment by activatedgeek** [comment]
Also created an SO post [here](http://stackoverflow.com/questions/30215933/unable-to-get-juju-working-on-azure)

**Comment by anastasiamac** [comment]
This issue has been fixed on 1.25. 
Juju 2.0 uses newer Azure SDK.

We track our bugs in launchpad: https://bugs.launchpad.net/juju/
